index.bs

```diff
@@ -1767,6 +1786,12 @@ a [=source type=] |sourceType|, and a [=duration=] |expiry|:
 1. If |map|["`event_report_windows`"] does not [=map/exist=], return the result
     of [=obtaining default effective windows=] given |sourceType|, |sourceTime|,
     and |expiry|.
+1. Return the result of [=parsing report windows ignoring legacy=] with |map|,
```

Comment:
It seems odd to use the word, "legacy." I think of a specification as always current. Also, in some time, we might have another feature that could have a concept of "legacy" and we may need to differentiate. If a feature is supported then it should have a descriptive name based on the feature.

Reply:
Do you have a suggestion?

Reply:
Something like `To parse a report window list`?

Reply:
`...event report window struct`?